### PR TITLE
added delete unit tests for pass and fail

### PR DESCRIPTION
**Pull request by NicholasHyland**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -156,7 +156,9 @@ def update_recommendations(id):
 def delete_recommendations(id):
     if get_recommendations(id).status_code == 200:
         conn.execute("DELETE FROM recommendations WHERE id=%d" % id)
-    return '', HTTP_204_NO_CONTENT
+        return '', HTTP_204_NO_CONTENT
```
Comment (0x932b) [inline server.py]: I vaguely remembered that the proffesor said delete requests should return 204 all the time?

Comment (sadam0930) [reply server.py]: I believe it was that DELETE is idempotent, so it should not return 404 on subsequent attempts. It should return 204.

Comment (sadam0930) [reply server.py]: @NicholasHyland can you please see our comments and push an update when you can?

**Review by sadam0930 — APPROVED** [review]
I made some fixes to #78 and #73

**Pull request merged**